src/jsonrpc/platformvm.rs

```diff
@@ -860,7 +857,6 @@ impl ApiPrimaryValidator {
             delegator_count: None,
             delegator_weight: None,
             delegators: None,
-            staked: None,
```

Comment:
Leaving a comment here to make sure your other PR is merged first. In the meantime, I think you can take this out of Draft!